yazi-plugin/preset/components/header.lua

```diff
@@ -14,6 +14,33 @@ function Header:cwd()
 	return span:style(THEME.manager.cwd)
 end
 
+function Header:counter()
+	local selected = #cx.active.selected
+	local yanked = #cx.yanked
+
+	local count
+	local style
+	if cx.yanked.is_cut then
+		count = yanked
+		style = THEME.manager.count_cut
+	elseif yanked > 0 then
+		count = yanked
+		style = THEME.manager.count_copied
+	else
+		count = selected
```

Comment:
```suggestion
		count = cx.active.selected
```

Reply:
If you want the least amount of variable declarations, should I also do `local count = #cx.yanked` and then update it to `selected` within the `if` block?

Reply:
Yeah, because accessing these variables also has a cost (albeit a small one), but it would be better if possible to put them in an if-else block.

Let me try to change it

Reply:
Ah so just moving `selected` and `yanked` into the `if` block should be what you want then? Just need to repeat the `count = #cx.yanked` line but that's fine

Reply:
yep i pushed a new commit should resolved this